install_dependencies.R

```diff
@@ -1,4 +1,8 @@
-install.packages(c('testthat', 'vdiffr', 'spelling','covr'), repos = 'http://cran.us.r-project.org')
+install.packages(c('testthat', 'vdiffr', 'spelling','covr','patchwork'), repos = 'http://cran.us.r-project.org')
```

Comment:
Why install patchwork while it is nor an Import or Suggest package ?

Reply:
Because: https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-R/branch/develop
![grafik](https://user-images.githubusercontent.com/25061876/231107871-4987df07-a2f9-4ea3-b215-00bf620b68e8.png)

Reply:
> Maybe installing the osp packages **with dependencies** 

Yes I think so.

Reply:
will try devtools::install_local() instead

Reply:
I stumbled on the same issue. `install_local()` should do it indeed even though I I have an error from RTools. But I think this is an issue with my configuration.

Reply:
I think we could come to the same result using `remotes::install_github("Open-Systems-Pharmacology/TLF-Library")` which installs the dependency and also targets the latest `develop` commit.